api/service.py

```
@@ -68,6 +85,19 @@ async def lifespan(app: FastAPI):
 
+    # Initialize observability (if available)
+    if OBSERVABILITY_ENABLED:
+        try:
+            logger = get_logger(__name__)
+            setup_json_logging(log_level=os.getenv("LOG_LEVEL", "INFO"))
+            initialize_tracing()
+            setup_auto_instrumentation()
+            instrument_fastapi(app)
+            startup_metrics_server(port=9090)
+            logger.info("event", "observability_initialized", service="astra-guard", version="1.0.0")
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)